charts/actions-runner-controller/templates/deployment.yaml

```diff
@@ -46,6 +46,9 @@ spec:
         - "--sync-period={{ .Values.syncPeriod }}"
         - "--docker-image={{ .Values.image.dindSidecarRepositoryAndTag }}"
         - "--runner-image={{ .Values.image.actionsRunnerRepositoryAndTag }}"
+        {{- if .Values.image.actionsRunnerPullImageSecretName }}
+        - "--runner-image-pull-secret-name={{ .Values.image.actionsRunnerPullImageSecretName }}"
```

Comment:
I'm a bit confused but you didn't actually define this command-line in the actions-runner-controller command yet?

Reply:
Addressed in 1f6facb775899f0b878b1c3a2e7f34fffe508242